### ISSUE TITLE
After latest update-- This site can’t be reached

### ISSUE THREAD

**Issue by tanglewood16** [teslamate-org/teslamate#3442]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I just upgraded to the latest release on my Raspberry Pi by doing the following:

1. Edit my image: postgres:13 -> 15.
2. Pulldown: $ docker compose pull
3. Run: $ docker compose up -d

Alas, when I access TeslaMate in my browser at 'raspberrypi:4000', I get

"This site can't be reached."

Here are a few things I have tried/observed:
1. I tried rebooting the pi and that didn't make a difference.
2. I am able to ssh into the pi.
3. Interestingly, raspberrypi:3000 does respond, just that there's 'no data' in those charts and graphs.
4. When I do
       $ ps auxwww | grep 3000
    I see a couple of processes doing docker-proxy with host-ip and container-ip to 3000.
5. When I do
       $ ps auxwww | grep 4000
    nothing turns up.

It seems to me that the piece that's supposed to listen to 4000 isn't running in my set up. Any idea how I can fix this? Thanks!

### Relevant log output

```shell
pi-teslamate-1  | waiting for postgres at database:5432

repeating 815 times.
```

### Type of installation

Docker

### Version

v1.27

**Comment by damienbr** [comment]
Same behavior here. Since the last version, the container shutdown without any logs and keeps restarting.
```
teslamate_1  | 2023-11-17 08:17:04.855 [info] Migrations already up
teslamate_1  | 2023-11-17 08:17:11.032 [info] System Info: Erlang/OTP 26 (jit)
teslamate_1  | 2023-11-17 08:17:11.032 [info] Version: 1.27.4
teslamate_1  | 2023-11-17 08:17:12.109 [info] POST https://auth.tesla.com/oauth2/v3/token -> 200 (979.656 ms)
teslamate_1  | 2023-11-17 08:17:12.109 [info] Refreshed api tokens
teslamate_1  | 2023-11-17 08:17:12.120 [info] Scheduling token refresh in 6 h
teslamate_1  | 2023-11-17 08:17:12.122 [info] Running TeslaMateWeb.Endpoint with cowboy 2.10.0 at :::4000 (http)
teslamate_1  | 2023-11-17 08:17:12.124 [info] Access TeslaMateWeb.Endpoint at http://localhost
teslamate_1  | 2023-11-17 08:17:12.320 [info] Starting logger for 'Elise et Damien'
teslamate_1  | 2023-11-17 08:17:12.342 [info] MQTT connection has been established
teslamate_1  | 2023-11-17 08:17:12.557 car_id=1 [info] Start / :asleep
teslamate_1  | 2023-11-17 08:17:14.716 [info] tzdata release in place is from a file last modified Fri, 22 Oct 2021 02:20:47 GMT. Release file on server was last modified Tue, 28 Mar 2023 20:25:39 GMT.
```

It ended up working after a couple of crashes / restarts...
It would be nice to have the log level configurable via an environment variable.

**Comment by JakobLichterfeld** [comment]
You need to follow dedicated instructions when updating postgres version, see https://docs.teslamate.org/docs/maintenance/upgrading_postgres

**Comment by damienbr** [comment]
I did it Jakob, but it still crashes.
```

teslamate_1  | 2023-11-17 08:45:54.861 [info] Migrations already up
teslamate_1  | 2023-11-17 08:46:02.267 [info] System Info: Erlang/OTP 26 (jit)
teslamate_1  | 2023-11-17 08:46:02.267 [info] Version: 1.27.4
teslamate_1  | 2023-11-17 08:46:03.161 [info] POST https://auth.tesla.com/oauth2/v3/token -> 200 (765.942 ms)
teslamate_1  | 2023-11-17 08:46:03.161 [info] Refreshed api tokens
teslamate_1  | 2023-11-17 08:46:03.168 [info] Scheduling token refresh in 6 h
teslamate_1  | 2023-11-17 08:46:03.170 [info] Running TeslaMateWeb.Endpoint with cowboy 2.10.0 at :::4000 (http)
teslamate_1  | 2023-11-17 08:46:03.172 [info] Access TeslaMateWeb.Endpoint at http://localhost
teslamate_1  | 2023-11-17 08:46:03.376 [info] Starting logger for 'xxx'
mosquitto_1  | 1700210763: New connection from 172.19.0.3:55670 on port 1883.
mosquitto_1  | 1700210763: New client connected from 172.19.0.3:55670 as TESLAMATE_302E3738323 (p2, c1, k60).
teslamate_1  | 2023-11-17 08:46:03.385 [info] MQTT connection has been established
teslamate_1  | 2023-11-17 08:46:03.481 car_id=1 [info] Start / :asleep
teslamate_1  | 2023-11-17 08:46:06.084 [info] tzdata release in place is from a file last modified Fri, 22 Oct 2021 02:20:47 GMT. Release file on server was last modified Tue, 28 Mar 2023 20:25:39 GMT.
mosquitto_1  | 1700210767: Client TESLAMATE_302E3738323 closed its connection.
teslamate_1  | 2023-11-17 08:46:12.915 [info] Migrations already up
teslamate_1  | 2023-11-17 08:46:18.935 [info] System Info: Erlang/OTP 26 (jit)
teslamate_1  | 2023-11-17 08:46:18.935 [info] Version: 1.27.4
teslamate_1  | 2023-11-17 08:46:19.458 [info] POST https://auth.tesla.com/oauth2/v3/token -> 200 (397.123 ms)
teslamate_1  | 2023-11-17 08:46:19.458 [info] Refreshed api tokens
teslamate_1  | 2023-11-17 08:46:19.468 [info] Scheduling token refresh in 6 h
teslamate_1  | 2023-11-17 08:46:19.470 [info] Running TeslaMateWeb.Endpoint with cowboy 2.10.0 at :::4000 (http)
teslamate_1  | 2023-11-17 08:46:19.472 [info] Access TeslaMateWeb.Endpoint at http://localhost
teslamate_1  | 2023-11-17 08:46:19.642 [info] Starting logger for 'xxx'
teslamate_1  | 2023-11-17 08:46:19.655 [info] MQTT connection has been established
mosquitto_1  | 1700210779: New connection from 172.19.0.3:60054 on port 1883.
mosquitto_1  | 1700210779: New client connected from 172.19.0.3:60054 as TESLAMATE_302E3631373 (p2, c1, k60).
teslamate_1  | 2023-11-17 08:46:19.759 car_id=1 [info] Start / :asleep
mosquitto_1  | 1700210784: Client TESLAMATE_302E3631373 closed its connection.
damien_teslamate_1 exited with code 137

```

**Comment by damienbr** [comment]
It seems to be related to memory consumption.
I increased the RAM from 2GB to 4GB and it seems stable now.
Did you expect an increase of the memory footprint?

**Comment by JakobLichterfeld** [comment]
> It seems to be related to memory consumption. I increased the RAM from 2GB to 4GB and it seems stable now. Did you expect an increase of the memory footprint?

No, still running on Raspberry Pi Model 3 B with 1 GB RAM without any issues even with other docker containers in parallel.

**Comment by JakobLichterfeld** [comment]
> I did it Jakob, but it still crashes.
My comment was in reference to OP not following the appropriate steps.
